src/style.css

```
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+}
+
+.stars-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.star {
+  background: url(images/star.svg) no-repeat center ;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.star5 {
+  margin-right: 0;
+}
+
+.stars-wrapper:nth-child(2) > .star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars-wrapper:nth-child(3) > .star:nth-child(1),
+.stars-wrapper:nth-child(3) > .star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars-wrapper:nth-child(4) > .star:nth-child(1),
+.stars-wrapper:nth-child(4) > .star:nth-child(2),
+.stars-wrapper:nth-child(4) > .star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars-wrapper:nth-child(5) > .star:not(:last-child),
+.stars-wrapper:nth-child(6) > div {
+  background-image: url(images/star-active.svg);
+}
```

Try to understand this solution

```suggestion
.stars--1 star:first-child,
.stars--2 star:nth-child(-n+2),
.stars--3 star:nth-child(-n+3),
.stars--4 star:nth-child(-n+4),
.stars--5 star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```